webapp/channels/src/components/admin_console/admin_settings.tsx

```diff
@@ -160,7 +160,7 @@ export default abstract class AdminSettings <Props extends BaseProps, State exte
         return n;
     };
 
-    private parseIntNonNegative = (str: string, defaultValue?: number) => {
+    public parseIntNonNegative = (str: string, defaultValue?: number) => {
```

Comment:
As mentioned in this 

We can allow the str param to be either a string or number and subsequently, in the function body check if its a string, parse it to number or if its a number take it as it is